drake/solvers/NPCPP/BoundingBoxConstraint.cpp

```diff
@@ -0,0 +1,12 @@
+#include "BoundingBoxConstraint.h"
```

Comment:
I would tend to put many of the constraints into single files, Contraint.h and Constraint.cpp.  Splitting up files is good, but not when they're this simple??